backend/user-service/src/main/java/hu/me/iit/malus/thesis/user/service/impl/UserServiceImpl.java

```diff
@@ -240,28 +160,33 @@ public void saveCourseCreation(String teacherEmail, Long courseId) {
      */
     @Transactional
     @Override
-    public void saveCourseAssign(String studentEmail, Long courseId) {
-        Optional<Student> optionalStudent = studentRepository.findLockByEmail(studentEmail);
-        if (optionalStudent.isPresent()) {
+    public void assignStudentsToCourse(Long courseId, List<String> studentEmails) {
+        Mail mail = new Mail();
```

Comment:
Ez ha nem is kulon http keres (email uService), de akkor is kulon szerviz lesz.

Reply:
Nem értem hogy gondolja, a usereknek van egy listája hogy milyen courseokhoz vannak hozzárendelve, abba rakja bele ez a course idt és userek email címe az idjük azért kell itt hogy mely usernek kell a listájába rakdni a courseId-t. Email már az email szerviz küldi itt csak összerakjuk a dto-t neki.